styleguide/_themes/global/scss/grid.scss

```diff
@@ -16,3 +26,12 @@
     width: $container-lg;
   }
 }
+
+.container,
+.container-fluid {
+  position: relative;
```

Comment:
So I'm a little nervous about adding this `position: relative` here. I don't think it's gonna affect anything, unless we're using absolute positioning relative to the document root anywhere.  The purpose of it, was to give me an anchor point for our anchored text behavior in zoolander, but I guess we probably need to do some thorough testing of www.